css/ISO50_style.css

```diff
@@ -0,0 +1,96 @@
+body{
+  /*background-color: #FFFFF0;*/
+  font-family: 'Nunito Sans', sans-serif;
+  font-weight: 300;
+  margin: 0px 0px;
+  line-height: 1.5;
+  font-size: 14px;
+}
+
+h1{
+  padding-top: 25px;
+  text-transform: uppercase;
+  font-weight: 300;
+  margin-bottom: 0px;
+  font-size: 40px;
+  letter-spacing: 1.5px;
+}
+
+p{
+  font-weight: 300;
```

Comment:
since the `font-weight` is declared in the body tag. That declaration will inherit down the stylesheet unless it is  declared with another value in another selector further down in the style sheet.